### ISSUE TITLE
How to change the Viewer's preview quality to prevent slowness

### ISSUE THREAD

**Issue by Cherryfrin** [tahoma2d/tahoma2d#623]
I use a macOS High Sierra 10.13.6 and a huion430p tablet to draw.Im currently working on an animation with 100+ levels all tidied up into sub-scenes.
I think I'm at fault for loading so many layers but the preview when I play it is very slow.

**Comment by artisteacher** [comment]
Try using the flipbook instead for previewing -> CMD+R

**Comment by DarrenTAnims** [comment]
It plays slowly for everyone in the main window, for anything but a simple test. Using the preview window is the best way to test the timing of an animation.

**Comment by Cherryfrin** [comment]
Yes since with the preview window you can also follow the timing of the XSheet.I think maybe I should configure my Preferences?
